compiler/src/dmd/frontend.h

```diff
@@ -3796,7 +3796,6 @@ class FuncDeclaration : public Declaration
     Array<Expression* >* fdrequireParams(Array<Expression* >* param);
     Array<Expression* >* fdensureParams(Array<Expression* >* param);
     FuncDeclaration* syntaxCopy(Dsymbol* s) override;
-    bool functionSemantic3();
```

Comment:
Note that this function is still part of the C++ interface, it's just that it's a global function now. That's why circle ci is complaining.

Reply:
to update frontend.h just use build.d from compiler/src: `./build.d cxx-headers-test AUTO_UPDATE=1`